<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceData_Online</class>
 <widget class="QWidget" name="DeviceData_Online">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}
QLabel
{
color:white;
}</string>
  </property>
  <widget class="QLabel" name="LBL_GW1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GW1</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_GW2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GW2</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_RIOM1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RIOM1</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_DDU">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DDU1</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_TCU2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TCU2</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_TCU1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TCU1</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_MPU1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TCU1</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_MPU2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TCU2</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_RIOM2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RIOM2</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_GW3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TCU2</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_TCU3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>190</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TCU3</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_TCU4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TCU4</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_ACU2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>260</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TCU4</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_ACU1">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>190</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TCU3</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_CMD">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>400</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>6A/CMD</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_BCU">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>BCU</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_WTB_A">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>600</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,255,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_WTB_B">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>600</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,255,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
